guides/installation_guides/c++/mac/README.md

```diff
@@ -1,5 +1,34 @@
-# Cosmos Guides
-> Your personal library of every algorithm and data structure code that you will ever encounter
+# Cosmos Guide: Getting Ready to program in C++ in MAC OS
+<b1> <b>Getting The Software</b> </b1> <br> <br>
```

Comment:
What  is the `<b1>` tag? Also, for bolding, you can put the text inside two stars: `**text**`.